common/app/com/commercetools/sunrise/common/SunriseSolutionInfo.java

```diff
@@ -0,0 +1,13 @@
+package com.commercetools.sunrise.common;
+
+import io.sphere.sdk.client.SolutionInfo;
+import io.sphere.sdk.meta.BuildInfo;
+
+public class SunriseSolutionInfo extends SolutionInfo {
+    public SunriseSolutionInfo() {
+        setName("");
+        setVersion(BuildInfo.version());
+        setWebsite("");
+        setEmergencyContact("");
```

Comment:
Do we need to explicitly set the website and contact as empty strings? If possible, let's avoid calling the setters if we are not going to set anything.